src/components/front-office/welcome/presentation/Presentation.scss

```
@@ -0,0 +1,9 @@
+.pres-card {
+    border: none;
+    margin: 20px 100px 20px 100px ;
```

Syntaxe "short-hand" : les 4 ne sont pas nécessaires
```suggestion
    margin: 20px 100px;
```